janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/resources/cql-read.properties

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
+gremlin.hadoop.graphReader=org.janusgraph.hadoop.formats.cql.CqlInputFormat
+gremlin.hadoop.graphWriter=org.apache.hadoop.mapreduce.lib.output.NullOutputFormat
+gremlin.hadoop.jarsInDistributedCache=true
+gremlin.hadoop.inputLocation=none
+gremlin.hadoop.outputLocation=target/output
+
+janusgraphmr.ioformat.conf.storage.backend=cql
+janusgraphmr.ioformat.conf.storage.cql.keyspace=cqlinputformatit
+janusgraphmr.ioformat.conf.storage.hostname=localhost
```

Comment:
It would be worthwhile to define `janusgraphmr.ioformat.conf.storage.port=9042` here to reinforce the default port. Also would be helpful to create a `read-cql.properties` that would be packaged with the distribution in the `conf/hadoop-graph/` directory alongside the other OLAP examples.

Reply:
Good suggestion of course. I just pushed a commit for that.